.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -118,110 +113,78 @@ jobs:
         set -x
         df -h
         for delete in /usr/share/dotnet /usr/local/lib/android /opt/ghc /opt/hostedtoolcache/CodeQL; do
-          rm -rf "${delete:?}"
+          rm -rf "/mnt${delete:?}"
```

Comment:
Confused, why?

Reply:
When running in a container the host directories are mounted with the prefix `/mnt`.